xgboost_ray/examples/simple_ray_dataset.py

```diff
@@ -22,16 +24,7 @@ def main(cpus_per_actor, num_actors):
     data.columns = [str(c) for c in data.columns]
     data["label"] = y
 
-    # There was recent API change - the first clause covers the new
-    # and current Ray master API
-    if hasattr(ray.data, "from_pandas_refs"):
-        # Generate Ray dataset from 4 partitions
-        ray_ds = ray.data.from_pandas(data).repartition(num_actors)
-    else:
-        # Split into 4 partitions
-        partitions = [ray.put(part) for part in np.split(data, num_actors)]
-        ray_ds = ray.data.from_pandas(partitions)
-
+    ray_ds = ray.data.from_pandas(data).repartition(num_actors)
```

Comment:
do we even need to do repartition anymore? this should be handled inside xgboost-ray nowadays, no?

Reply:
You're right - removed